doc/panes/editor.rst

```diff
@@ -2,22 +2,37 @@
 Editor
 ######
 
-Spyder's multi-language **Editor** integrates a number of powerful tools right out of the box for an easy to use, efficient editing experience.
-The Editor's key features include syntax highlighting (``pygments``); real-time code and style analysis (``pyflakes`` and ``pycodestyle``); on-demand completion, calltips and go-to-definition features (``rope`` and ``jedi``); a function/class browser, horizontal and vertical splitting, and much more.
+Spyder's multi-language **Editor** pane is the key element of the IDE where users can create, open, and modify code files.
+The Editor offers a variety of core features like autocompletion, real-time analysis, syntax highlighting, horizontal and vertical splitting, and much more.
+In addition, it uniquely integrates a number of powerful tools for an easy to use, efficient editing experience.
 
 .. image:: /images/editor/editor-standard.png
    :alt: Spyder's Editor panel, split horizontally and with style analysis
 
-:guilabel:`Outline Explorer` (function/class/method browser) and horizontal/vertical splitting capabilities:
+===========================
+The Editor's key components
+===========================
 
-.. image:: /images/editor/editor-outline-standard.png
-   :alt: Spyder outline panel, showing the functions/classes/methods in a file
+The editor pane consists of the following areas:
 
-Real-time code and style analysis with ``pyflakes`` and ``pycodestyle``:
+.. image:: /images/editor/editor-components.png
+   :alt: Spyder's Editor pane, showing its different areas (described below)
 
-.. image:: /images/editor/editor-inset-code-analysis.png
-   :alt: A snippit of code in the Spyder Editor, showing code style warnings
+1. The left sidebar shows line numbers and displays any code analysis errors/warnings that exist in the current file.
+   Clicking a line number highlights the code in that line.
 
+.. image:: /images/editor/editor-code-error.png
+   :alt: Spyder Editor, showing an example of a code error popup message
+
+2. The right and bottom scrollbars help navigate the code in the current file vertically and horizontally.
+3. The Context menu displays actions relevant to the editor pane.
+   To display the editor's context menu, right-click anywhere over it and a list of possible options will appear under the cursor.
+4. Another menu called the Options menu appears when clicking the "Hamburger" icon at the top right, which includes useful settings and actions relevant to the editor.
```

Comment:
```suggestion
4. The options menu ("Hamburger" icon at top right) includes useful settings and actions relevant to the Editor.
```

* Revise phrasing to be simpler, more concise and easier to read
* The others all lead with "The [thing name] [what it does]", so frame this the same way
* The "options menu" is lowercase, by convention (we don't use Proper Case unless its actually named such in the GUI), while Editor is Proper Case